.gitignore

```diff
@@ -51,4 +51,5 @@ _NCrunch_*
 # Rider auto-generates .iml files, and contentModel.xml
 **/.idea/**/*.iml
 **/.idea/**/contentModel.xml
-**/.idea/**/modules.xml
\ No newline at end of file
+**/.idea/**/modules.xml
```

Comment:
we should get rid of all of this and just add `.idea`

Reply:
That's not the same thing. This is the recommended gitignore from jetbrains